Add pdfmake version 0.1.28

jeremyfourna opened the pull request:
Hello,

I created this pullrequest to update to the last version of pdfmake the meteor package.

I just replaced the files in the build directory by the last version available in pdfmake repo

Do I need to bump the version number of the meteor package also ?

Regards,

Jeremy

NilsDannemann commented:
Hey Jeremy.
Thanks for the PR.
Yes, I think it would be a good idea to bump the version of the meteor package as well.

Then I'll merge it.

jeremyfourna commented:
Hello @NilsDannemann,

I added a commit where I updated the package version, and added the new files that the original package now have (js map file)

Regards,

Jeremy

NilsDannemann commented:
Thanks!
